bsdlib/README.rst

```diff
@@ -3,9 +3,131 @@
 BSD Library
 ###########
 
+The BSD library is Nordic Semiconductor's implementation of the BSD Socket API, which is a set of standard function calls that can be used in an application.
+This library aims to be compatible with the BSD socket standard to simplify writing and porting of existing applications. 
+
+The library has the following set of sockets:
+
+* 8 generic sockets that can be used generically between UDP, TCP, TLS, DTLS, and AT commands.
+* 1 GNSS/AGPS socket
+* 1 Modem DFU socket
+* 10 PDN sockets
+
+Overview
+********
+
+.. figure:: doc/images/bsd_lib_overview.svg
+   :alt: BSD library overview diagram
+
+   BSD library overview diagram
+
+The BSD library is the primary interface to operate the nRF9160 modem in order to establish LTE-M, NBIOT, and GNSS connections.
+The library abstracts away the details on memory management and synchronization leaving the application to focus on writing the application using standardized API calls.
```

Comment:
leaving the application to focus on writing the application?

Reply:
The library abstracts away memory management details and application synchronization, leaving the developer to focus on writing the application using standardized API calls.

Reply:
Changed to:
'The library abstracts away the details on memory management and synchronization allowing you to focus on writing the application using standardized API calls.'